pyndustri.pyi

```diff
@@ -362,6 +362,16 @@ class Unit(Senseable):
         Note that horizon must be in movement for it to shoot.
         """
     @staticmethod
+    def target(self, x: int = None, y: int = None, shoot: bool = False):  # target
+        """
+        Command the unit to target (rotate to) the specified coordinates. If shoot is set to True, it will behave same as shoot(x,y)
```

Comment:
```suggestion
        Command the unit to target (rotate to) the specified coordinates.
        If shoot is set to `True`, it will behave same as `shoot(x, y)`.
```